src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -653,6 +653,43 @@ std::vector<std::pair<CNNLayerPtr, int> > CNNNetGetPrevLayersSkip(CNNLayerPtr or
     return prevLayers;
 }
 
+/**
+ * @brief Removes 'to_remove' layer from between two other - 'prev' and 'next', then connects 'prev' and 'next'
+ * param prev           Layer before 'to_remove'
+ * param to_remove      Layer to be removed
+ * param next           Layer next to 'to_remove'
+ * param prevOutputNo   Output number of 'prev', which will be connected with 'next'
+ * param nextInputNo    Input number of 'next', which will be connected with 'prev'
+ */
+inline bool CNNRemoveAndConnect(CNNLayerPtr prev, CNNLayerPtr to_remove, CNNLayerPtr next,
```

Comment:
if to_remove layeri is between prev and next, cannot we just retrieve this pointers from to_remove.

Reply:
Yes. It works.